members/S000033.yaml

```diff
@@ -180,17 +83,68 @@ contact_form:
           "Senior Airman": "Senior Airman"
           "Airman First Class": "Airman First Class"
           "Airman": "Airman"
-    - check:
-      - name: response
-        selector: "#response_y"
-        value: Yes
+    - find:
+      - selector: "#wpcf7-f8521-p20941-o1 input[name='FirstName']"
+    - fill_in:
+      - name: First Name
+        selector: "#wpcf7-f8521-p20941-o1 input[name='FirstName']"
+        value: "$NAME_FIRST"
+        required: true
+        options:
+          max_length: 40
+      - name: Last Name
+        selector: "#wpcf7-f8521-p20941-o1 input[name='LastName']"
+        value: "$NAME_LAST"
+        required: true
+        options:
+          max_length: 40
+      - name: Street Address
+        selector: "#wpcf7-f8521-p20941-o1 input[name='Address1']"
+        value: "$ADDRESS_STREET"
+        required: true
+        options:
+          max_length: 40
+      - name: Street Address2
+        selector: "#wpcf7-f8521-p20941-o1 input[name='Address2']"
+        value: "$ADDRESS_STREET_2"
+        required: false
+        options:
+          max_length: 40
+      - name: City
+        selector: "#wpcf7-f8521-p20941-o1 input[name='City']"
+        value: "$ADDRESS_CITY"
         required: true
+        options:
+          max_length: 40
+      - name: zipCode
+        selector: "#wpcf7-f8521-p20941-o1 input[name='ZipCode']"
+        value: "$ADDRESS_ZIP5"
+        required: true
+      - name: phone number
+        selector: "#wpcf7-f8521-p20941-o1 input[name='Telephone']"
+        value: "$PHONE"
+        required: false
+        options:
+          max_length: 40
+      - name: email 
+        selector: "#wpcf7-f8521-p20941-o1 input[name='your-email']"
+        value: "$EMAIL"
+        required: true
+        options:
+          max_length: 40 
+      - name: message
+        selector: "#wpcf7-f8521-p20941-o1 textarea[name='your-message']"
```

Comment:
These IDs look a bit autogenerated and subject to change... guessing we need them though?

Reply:
yeah I tried using other selectors but wasn't getting the specific elements I wanted